web/src/utils/RestApiClient.js

```diff
@@ -31,4 +31,24 @@ export default {
   getRequestList() {
     return RestApiClient.get('/api/request/summary')
   },
+
+  // Request Output
+  getRequestOutput(request_id) {
+    return RestApiClient.get('/api/result/request/' + request_id, { responseType: 'blob' })
+  },
+
+  // Task List
+  getTaskList(request_id) {
+    return RestApiClient.get('/api/request/' + request_id)
```

Comment:
Depending on the webserver config later, you probably want to append the trailing `/` to avoid redirects.

```suggestion
    return RestApiClient.get('/api/request/' + request_id + '/')
```

Reply:
same for other requests in this file

Reply:
ah good call, currently breaks the requests but FYI @jleaniz can file an issue for you unless this is already being addressed in the `api-client` PR you're working on

Reply:
ack, this can be followed up on later

Reply:
Chatting with @jleaniz was a little confused, is it to double check that the server config accepts a trailing `/` or not and update request query if so or to update web server to accept `/`  as best practice to avoid some sort of redirect weirdness?

Current behavior:
```
INFO:     [127.0.0.1:34520](http://127.0.0.1:34520/) - "GET /api/request/ebac3daee8e04b67be86abcd9e462aa8/ HTTP/1.1" 307 Temporary Redirect
INFO:     [127.0.0.1:34520](http://127.0.0.1:34520/) - "GET /api/request/ebac3daee8e04b67be86abcd9e462aa8 HTTP/1.1" 200 OK
```